### ISSUE TITLE
Problem: nothing provides package_has_been_revoked needed by gfortran_linux-64-7.2.0-24

### ISSUE THREAD

**Issue by cs224** [conda-forge/aesara-feedstock#22]
I am trying to install aesara via [mamba](https://github.com/mamba-org/mamba), because conda is so slow that I cannot wait for it.

Whenever I try to install aesara I get the error message:

    > mamba install -c conda-forge aesara
    Encountered problems while solving.
    Problem: nothing provides package_has_been_revoked needed by gfortran_linux-64-7.2.0-24

I do not understand, because

    mamba repoquery depends aesara -c conda-forge -t

gives me the dependecy tree and there is nowhere a dependency on the gnu compiler suite 7.2.0. All dependencies are on 7.3.0.

I tried the install into a frech environment with `conda` itself, but stopped the process after waiting for 30 minutes.

Any ideas?

Thanks a lot!
Christian

**Comment by brandonwillard** [comment]
I just ran the following without issue (on Ubuntu 20.04.2):
```
> mamba --version
mamba 0.14.1
conda 4.10.3
> mamba clean --all --force-pkgs-dirs
> HOME=$(mktemp -d) mamba create -n aesara-testing -y --no-default-packages -c conda-forge aesara
> conda activate aesara-testing
> python -c "import aesara; print(aesara.__version__)"
2.0.12
```

The `HOME=$(mktemp -d)` is there to prevent the use of my local `~/.condarc`.  Out of an abundance of caution, I added `--no-default-packages` as well.

Perhaps there's something about your Conda/shell environment that's causing this issue.

**Comment by cs224** [comment]
@brandonwillard : thank you very much! I created a completely fresh anaconda environment from scratch, only installed mamba and then tried to install aesara with my above described results. Thanks to your detailed description above, I found out that while I had conda 4.10.3, I had mamba version 0.7.6. I was able to upgrade it to 0.14.0 (v. 0.14.1 depends on python 3.9, which is not the default in a fresh installed anaconda environment) and after the upgrade it works now.

Thanks a lot!!
Christian
